### PR TITLE
PHP 7.2: Update OneLogin SAML dependency, use SS_Object

### PR DESCRIPTION
**Pull request by lukereative**

### CODE REVIEW

**Comment by brynwhyman** [comment]
Hey @lukereative thanks for the PR! We've merged similar changes already (https://github.com/silverstripe/silverstripe-realme/pull/45). Check out the `master` branch to see if there's still more you'd like to commit.

We're currently holding off on a new release until we can complete testing the RealMe 'Assert' verification type. At this stage we've seen successful testing of the RealMe 'Login' verification only.

**Comment by lukereative** [comment]
I see that, this is for the v2 branch, because I need compatibility with SS3 and it seemed like SS was unsure when they were going to get to doing combat for v2 of this module.

**Comment by brynwhyman** [comment]
Ah! Sorry I missed that... We'd welcome any support in getting the SS3 version over the line. I've linked your PR to https://github.com/silverstripe/silverstripe-realme/issues/24.

Any notes you can add to that issue on any subsequent work or testing you've done would be really helpful!

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "require": {
         "composer/installers": "*",
         "silverstripe/framework": "~3.1",
```
Comment (NightJar) [inline composer.json]: Could you please also update the framework requirement to `^3.7`?
`SS_Object` isn't a thing in earlier versions :)

Comment (lukereative) [reply composer.json]: True, missed that since my project had 3.7 as a req.

**Comment by lukereative** [comment]
@NightJar You're welcome. I've fixed the composer req. and changed to `use` statements. I've tested it locally for login auth using MTS on a SS3 project that we're doing a recipe upgrade for – seems to work fine so far.

**Comment by NightJar** [comment]
Thanks @lukereative !

**Pull request merged**